src/geo/projection/globe.js

```diff
@@ -104,25 +104,42 @@ export default class Globe extends Mercator {
     }
 
     pointCoordinate(tr: Transform, x: number, y: number, _: number): MercatorCoordinate {
```

Comment:
Just a note on a somewhat related topic I noticed while testing this PR, screen points not falling on the globe are currently returned as valid longitude/latitude when using `map.unproject(...)`

https://user-images.githubusercontent.com/7061573/157287561-ab95c266-293e-4174-a047-2c6b8e2119b3.mov

I'll open a ticket about that because these projection API need special attention.

Reply:
thanks!